modules/iam-primary-roles/main.tf

```diff
@@ -48,7 +50,7 @@ data "aws_iam_policy_document" "saml_provider_assume" {
       type = "Federated"
 
       # Loop over the IDPs from the `sso` component
-      identifiers = [for name, arn in data.terraform_remote_state.sso.outputs.saml_provider_arn : arn]
+      identifiers = [for name, arn in data.terraform_remote_state.sso.outputs.saml_provider_arns : arn]
```

Comment:
Was this a bug that you're fixing?

Reply:
@Gowiem yes, see here https://github.com/cloudposse/terraform-aws-components/blob/master/modules/sso/outputs.tf#L1